### PR TITLE
Repeat is back...

### PR DESCRIPTION
**Pull request by callumok2004**
works like a charm
https://i.callumdev.pw/e2f1y.png
https://i.callumdev.pw/8yxf8.png

### CODE REVIEW

**Review by Bas950 — DISMISSED** [review]

File: websites/R/Repeat/dist/metadata.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "repeatradio.net",
     "repeat.pw"
   ],
-  "version": "1.0.2",
-  "logo": "https://i.imgur.com/Mh7vi7e.png",
+  "version": "2.0.0",
+  "logo": "https://i.imgur.com/I5G1Lvn.png",
```
Comment (ririxidev) [inline websites/R/Repeat/dist/metadata.json:16]: 512x512

File: websites/R/Repeat/presence.ts [hunk 2]
```diff
@@ -1,33 +1,43 @@
...
-
-presence.on("UpdateData", () => {
-  const presenceData: PresenceData = {
-    largeImageKey: "repeatlogo",
-    details: `Streaming to ${slisteners || 0} listeners`,
-    state: `${strack || "Title"} - ${sartist || "Artist"}`,
-    smallImageText: `${sdj || "AutoDJ"} is live!`,
-    startTimestamp: stamp
+presence.on("UpdateData", async () => {
+  if(!data) return;
```
Comment (ririxidev) [inline websites/R/Repeat/presence.ts:21]: if data maybe show something like idk error?

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/R/Repeat/dist/metadata.json (hunk 1 above)
Comment (callumok2004) [reply websites/R/Repeat/dist/metadata.json:16]: it is 512x512

Comment (ririxidev) [reply websites/R/Repeat/dist/metadata.json:16]: It's 400x400

File: websites/R/Repeat/presence.ts [hunk 3]
```diff
@@ -1,33 +1,43 @@
...
+  
+  const settings = {
+    details: (await presence.getSetting("details"))
+      .replace("%listeners%", data.listeners || "Listeners"),
+      state: (await presence.getSetting("state"))
+      .replace("%artist%", data.song.artist || "Artist")
+      .replace("%songText", data.song.text || "Song")
+      .replace("%title%", data.song.title || "Title"),
+    timestamp: await presence.getSetting("timestamp")
```
Comment (Bas950) [inline websites/R/Repeat/presence.ts:30]: why return? You have fallback strings anyway?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by callumok2004** [comment]
reeeeeeeeeeeee

**Pull request closed without merging**